pre-start.d/01-remove_unused_packages.sh

```diff
@@ -30,6 +30,8 @@ removed_packages="python-alembic
                   xivo-ctid
                   xivo-ctid-client
                   xivo-ctid-client-python3
+                  xivo-agentd-cli
+                  xivo-agentd-client
```

Comment:
We cannot remove these package automatically. If user has script, then the script will be broken

Reply:
fixed